website/blueprints/hds-component-docs/files/docs/components/__name__/index.md

```diff
@@ -5,6 +5,7 @@ caption: {Short description that appears in the cards}
 links:
   figma: {link to the "<%= columnizedModuleName %>" page in the components Figma library}
   github: {link to the "<%= columnizedModuleName %>" component's folder in the GitHub repo}
+related: ['path/to/component', 'path/to/pattern']
```

Comment:
@cristiano Is this good? Is it ok to include an array here like this vs. `{...}`?

Reply:
yeah. in any case it will trigger an error if the developer forgets to update it (or remove it) 😀